bluesky/bundlers.py

```diff
@@ -293,15 +285,18 @@ async def _cache_describe(self, obj):
 
     async def _cache_describe_config(self, obj):
         "Read the object's describe_configuration and cache it."
-        if isinstance(obj, Configurable):
+
+        # Pylance doesn't understand this is an acceptable isinstance with the
+        # @runtime_checkable, hence the type ignore.
+        if isinstance(obj, Configurable):  # type: ignore
```

Comment:
It would require using a 
``` python
# pyright: ignore [X]
```

where `X` is one of [these options](https://github.com/microsoft/pyright/blob/main/docs/configuration.md). I've tried searching for it but I can't find which of these this error belongs to...

Reply:
The latest update of vscode doesn't seem to error on these lines for me even without the ignore